Include/object.h

```diff
@@ -304,6 +304,7 @@ PyAPI_FUNC(int) PyObject_SetAttr(PyObject *, PyObject *, PyObject *);
 PyAPI_FUNC(int) PyObject_HasAttr(PyObject *, PyObject *);
 PyAPI_FUNC(PyObject *) PyObject_SelfIter(PyObject *);
 PyAPI_FUNC(PyObject *) PyObject_GenericGetAttr(PyObject *, PyObject *);
+PyAPI_FUNC(PyObject *) PyObject_GenericTryGetAttr(PyObject *, PyObject *);
```

Comment:
I don't think we should expose this to the public C API. Could you please add this to `Include/internal/pycore_object.h` instead? And remove the `PyAPI_FUNC` once it's there, `extern` is enough. Thanks!